packages/pds/src/api/com/atproto/server/requestAccountDelete.ts

```diff
@@ -18,7 +18,7 @@ export default function (server: Server, ctx: AppContext) {
         calcKey: ({ auth }) => auth.credentials.did,
       },
     ],
-    auth: ctx.authVerifier.accessCheckTakedown,
+    auth: ctx.authVerifier.accessFull({ checkTakedown: true }),
```

Comment:
yeah i'm not really sure this matters because ultimately you need a password to delete an account. But this request should really only be coming from a fully-permissioned session. I think we didn't the full check earlier because we didn't have an authVerifier helper to make it possible